Feature/fieldlist netcdf

JamesVarndell opened the pull request:
Adds `to_points` and `to_proj` methods to the NetCDFField class.

sandorkertesz commented:
Hi James, I noticed you have ifas.nc in docs, it is a huge file (22MB) and should not be stored in the repo. I created an emohawk space on get.ecmwf.int  https://get.ecmwf.int/#browse/browse:test-data:emohawk and put all the larger files there, and they can be downloaded on demand.

JamesVarndell commented:
Hi @sandorkertesz, thanks for the good point! I have uploaded the offending netCDF to the nexus repo and removed it from the GitHub repo - I've also added a method for retrieving remote data for testing purposes.
